core/src/main/groovy/noe/tomcat/configure/AjpConnectorTomcat.groovy

```diff
@@ -19,10 +19,6 @@ public class AjpConnectorTomcat extends ConnectorTomcatAbstract<AjpConnectorTomc
   private String secret
   private String allowedRequestAttributesPattern
 
-  public AjpConnectorTomcat() {
```

Comment:
I would leave protocol = "AJP/1.3" because this is in AjpConnectorTomcat class and there is no other option so far (AFAIK), si this is IMHO OK.

Reply:
It sure doesn't affect anything. Will put it back